install/kubernetes/helm/istio/values.yaml

```diff
@@ -113,6 +113,9 @@ global:
   # If not set, controller watches all namespaces
   oneNamespace: false
 
+  # istio control plane namespace
+  istioNamespace: istio-system
+
```

Comment:
? This doesn't seem right.. What is the .Release.namespace for ?
Also, why have this variable here when its not used by anything?

Put another way, if we install istio in a different namespace, how will someone know to update this variable as well?

Reply:
it is only used by the gateways to indicate so the gateway knows where the control plane runs.  (Today's current code hardcoded it to istio-system, with this it is configurable.). Yes, user does need to update this as well for very advanced installation where control plane is installed in a namespace other than istio-system.

Reply:
Release.namespace is where gateway is installed.
istioNamespace is where istio control plane is installed.
They may be different.

This is reusing the chart for ingress for installing ingress in different namespace.